staging/src/k8s.io/kube-scheduler/config/v1/types.go

```diff
@@ -65,6 +65,7 @@ type KubeSchedulerConfiguration struct {
 	// then scheduler stops finding further feasible nodes once it finds 150 feasible ones.
 	// When the value is 0, default percentage (5%--50% based on the size of the cluster) of the
 	// nodes will be scored.
+	// Overridden by per profile PercentageofNodesToScore if set.
```

Comment:
The comments in unversioned and versioned types.go are expected to be identical, could you help consolidate them?

Reply:
changed both to `It is overridden by profile level PercentageofNodesToScore if set`.